docs/user/quick-start.md

```diff
@@ -0,0 +1,80 @@
+
+## Provisioning a Control Plane Cluster with required resources
+
+You can provision a local k3d cluster as a control plane using the following command, by default, a cluster named `k3d-kyma` will be created. 
+```
+kyma provision k3d
+```
+Since the services deployed in the control plane are managed by Istio, you need to install Istio into the local k3d cluster using the following command:
+```
+istioctl install -y
+```
+Lifecycle Manager also exposes metrics that are collected by Prometheus Operator in the control plane to provide better observability. To simplify the local setup, you only need to deploy the ServiceMonitor CRD using the following command:
+```
+kubectl apply -f hack/servicemonitors.yaml
+```
+You can also follow official [quick start](https://prometheus-operator.dev/docs/prologue/quick-start/) guide to deploy a full set of prometheus operator into cluster as an alternative solution if you want to monitor the component performance.
+
+## Deploy Kyma lifecycle manager
+We recommend deploying Kyma Lifecycle Manager with the control plane kustomize profile, and `kcp-system`, `kyma-system` namespace must be configured upfront.
+```
+kubectl create ns kcp-system
+kubectl create ns kyma-system
+kyma alpha deploy -c alpha -k https://github.com/kyma-project/lifecycle-manager/config/control-plane
+```
+
+If the deployment is successful, you should see the following main resources:
+
+- Kyma Lifecycle Manager deployment instance named as `klm-controller-manager` under `kcp-system` namespace
+- All Module Templates configured in [kyma official repository](https://github.com/kyma-project/kyma/tree/main/modules) get deployed in `control-plane` as target
+- A Kyma CR that uses the global `alpha` channel but without any module configured, sync disabled, named `default-kyma` under `kyma-system` namespace
+
+### Manage modules in single cluster mode
+By default, all Module Templates deployed by Kyma CLI will be configured for single cluster mode (`spec.target=control-plane`), which means the module will be deployed in the Control Plane cluster, together with Kyma Lifecycle Manager.
+
+With Kyma CLI [enable module](https://github.com/kyma-project/cli/blob/main/docs/gen-docs/kyma_alpha_enable.md) and [disable module](https://github.com/kyma-project/cli/blob/main/docs/gen-docs/kyma_alpha_disable.md) command, you can manage kyma modules without directly modify Kyma CR.
+
+As an example, you can use following command to enable keda module in `alpha` channel for `default-kyma` under `kyma-system` namespace
+```
+kyma alpha enable module keda -c alpha -n kyma-system -k default-kyma
+```
+### Manage modules in remote cluster mode
+To allow Kyma Lifecycle Manager manages Kyma modules in remote cluster, two prerequisite must be fulfilled.
+
+1. For the module to be deployed remotely, the related Module Template must be configured as `remote` in `spec.target`.
```

Comment:
Maybe also add a kubectl patch command that allows you to do that in a one-liner

Reply:
I think for a quick start, it's better not to introduce too much manual modification to interrupt the flow, the command I put here is basically all necessary for the setup.

In the following apply kyma-project modules, I highlighted there the target is configured as `remote` already. And above cli deploy command, the target is `control-plane` (haven't implemented yet), so the user doesn't have to worry about this spec value during quick start. If later needs to modify this field, the choice is left to the user. what do you think?

Reply:
Understood. I think in this case you should probably reference single/dual cluster mode then into a different guide but that's probably in the reference docs that Im building up. LGTM for me then